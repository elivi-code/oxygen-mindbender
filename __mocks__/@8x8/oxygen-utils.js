export * from '@8x8/oxygen-utils';

const useFontSize = jest.fn(() => 16);

export { useFontSize };
