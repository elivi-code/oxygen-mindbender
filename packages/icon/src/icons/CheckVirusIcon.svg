<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
<path fill-rule="evenodd" clip-rule="evenodd" d="M3.00024 6C6.98394 5.43301 9.02178 4.99756 12.0002 2C14.9843 5.00244 17.0165 5.43301 21.0002 6C22.6718 6.23792 19.537 22 12.0002 22C4.53417 22 1.32863 6.23792 3.00024 6ZM4.637 8.36108C4.81187 9.81222 5.27777 11.6472 5.94051 13.3718C7.52257 17.4888 9.76751 20 12.0002 20C14.2651 20 16.512 17.4928 18.0851 13.3754C18.7431 11.653 19.203 9.81954 19.3726 8.36824C19.3973 8.15658 19.4148 7.96011 19.4255 7.78268C15.9884 7.21549 14.0912 6.50001 12.0001 4.73139C9.91164 6.49687 8.01147 7.21449 4.58224 7.7814C4.59369 7.95727 4.61177 8.15175 4.637 8.36108ZM8.12625 11H8.00024C7.44795 11 7.00024 11.4477 7.00024 12C7.00024 12.5523 7.44795 13 8.00024 13H8.12625C8.48803 14.4056 9.59463 15.5122 11.0002 15.874V16C11.0002 16.5523 11.448 17 12.0002 17C12.5525 17 13.0002 16.5523 13.0002 16V15.874C14.4058 15.5122 15.5124 14.4056 15.8742 13H16.0002C16.5525 13 17.0002 12.5523 17.0002 12C17.0002 11.4477 16.5525 11 16.0002 11H15.8742C15.5124 9.59439 14.4058 8.4878 13.0002 8.12602V8C13.0002 7.44772 12.5525 7 12.0002 7C11.448 7 11.0002 7.44772 11.0002 8V8.12602C9.59463 8.4878 8.48803 9.59439 8.12625 11ZM12.0002 14C13.1048 14 14.0002 13.1046 14.0002 12C14.0002 10.8954 13.1048 10 12.0002 10C10.8957 10 10.0002 10.8954 10.0002 12C10.0002 13.1046 10.8957 14 12.0002 14ZM13.0002 12C13.0002 12.5523 12.5525 13 12.0002 13C11.448 13 11.0002 12.5523 11.0002 12C11.0002 11.4477 11.448 11 12.0002 11C12.5525 11 13.0002 11.4477 13.0002 12Z" fill="#636363"/>
</svg>
