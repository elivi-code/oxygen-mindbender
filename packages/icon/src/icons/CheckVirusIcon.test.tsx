/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { icon as theme } from '@8x8/oxygen-constants';
import CheckVirusIcon from './CheckVirusIcon';

describe('<CheckVirusIcon>', () => {
  let wrapper;

  function setup(props = {}) {
    return renderer.create(<CheckVirusIcon theme={theme} {...props} />).toJSON();
  }

  it('should render correctly', () => {
    wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
/* eslint-enable */
