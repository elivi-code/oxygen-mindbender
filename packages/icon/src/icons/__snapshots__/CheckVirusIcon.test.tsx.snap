// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CheckVirusIcon> should render correctly 1`] = `
.c0 {
  height: 24px;
  width: 24px;
  min-height: 24px;
  min-width: 24px;
  -webkit-user-select: auto;
  -moz-user-select: auto;
  -ms-user-select: auto;
  user-select: auto;
  pointer-events: auto;
  fill: #1C2025;
}

<svg
  className="c0"
  color={null}
  disabled={false}
  role="presentation"
  selected={false}
  size={null}
  viewBox="0 0 24 24"
>
  <g>
    <path
      clipRule="evenodd"
      d="M3.0002 6c3.9837-.567 6.0216-1.0024 9-4 2.9841 3.0024 5.0163 3.433 9 4 1.6716.238-1.4632 16-9 16-7.466 0-10.6716-15.762-9-16ZM4.637 8.361c.1749 1.4512.6408 3.2862 1.3035 5.0108C7.5225 17.4888 9.7675 20 12.0002 20c2.2649 0 4.5118-2.5072 6.0849-6.6246.658-1.7224 1.1179-3.5559 1.2875-5.0072.0247-.2116.0422-.408.0529-.5855-3.4371-.5672-5.3343-1.2827-7.4254-3.0513-2.0885 1.7655-3.9886 2.483-7.4179 3.05.0115.1759.0296.3704.0548.5797ZM8.1263 11h-.126c-.5523 0-1 .4477-1 1s.4476 1 1 1h.126c.3617 1.4056 1.4683 2.5122 2.8739 2.874V16c0 .5523.4478 1 1 1 .5523 0 1-.4477 1-1v-.126c1.4056-.3618 2.5122-1.4684 2.874-2.874h.126c.5523 0 1-.4477 1-1s-.4477-1-1-1h-.126c-.3618-1.4056-1.4684-2.5122-2.874-2.874V8c0-.5523-.4477-1-1-1-.5522 0-1 .4477-1 1v.126C9.5946 8.4878 8.488 9.5944 8.1262 11Zm3.8739 3c1.1046 0 2-.8954 2-2s-.8954-2-2-2c-1.1045 0-2 .8954-2 2s.8955 2 2 2Zm1-2c0 .5523-.4477 1-1 1-.5522 0-1-.4477-1-1s.4478-1 1-1c.5523 0 1 .4477 1 1Z"
      fillRule="evenodd"
    />
  </g>
</svg>
`;
